### PR TITLE
Fix+pythonic style

### PR DESCRIPTION
**Pull request by Psycojoker**
Salut,

Un peu de nettoyage du code pour le rendre plus pythonic. À part le fix (j'ai fait ce qui me semblait évident) le comportement n'est pas modifié.

### CODE REVIEW

**Comment by M5oul** [comment]
Thanks. I will take a look.

File: package_linter.py [hunk 1]
```diff
@@ -132,7 +132,8 @@ def check_script(path, script_name):
     if check_file_exist(script_path) == 0:
         return
     print (c.BOLD + c.HEADER + "\n>>>>",
-           scripts[i].upper(), "SCRIPT <<<<" + c.END)
+           script_name.upper(), "SCRIPT <<<<" + c.END)
```
Comment (M5oul) [inline package_linter.py]: Thanks. Why it is working well?
It is a python cache?
Or the variable could be used on many function?

Comment (Psycojoker) [inline package_linter.py]: The variable `scripts` doesn't exist in this function, I just took the variable that seems to most appropriate here.

Comment (M5oul) [inline package_linter.py]: Yep, thanks.
But, it was working with `scripts[i]`.
That's strange.

Comment (Psycojoker) [inline package_linter.py]: It wasn't for me, I got an error there, you probably made some modifications in the mean time after it was working for you.

**Pull request merged**